client/my-sites/plans-features-main/hooks/use-is-free-domain-free-plan-upsell-enabled.ts

```diff
@@ -0,0 +1,19 @@
+import type { DataResponse } from 'calypso/my-sites/plans-grid/types';
```

Comment:
This is just a file movement. And a renaming of the function to reflect the fact that the universal state of the  `FreeDomainFreePlanUpsell` will be calculated here.